src/rpc/sync-error.cpp

```diff
@@ -154,6 +155,10 @@ QString SyncError::syncErrorIdToErrorStr(int error_id, const QString& path)
         return QObject::tr("%1 is not added to a library and will not be uploaded").arg(file);
     case SYNC_ERROR_ID_LIBRARY_TOO_LARGE:
         return QObject::tr("Library is too large to sync.");
+#if defined(Q_OS_WIN32)
+    case SYNC_ERROR_ID_MOVE_NOT_IN_REPO:
+        return QObject::tr("The moving source or destination directory is illegal");
```

Comment:
You can only move files/folders within a library or from one library to another library